conflux/src/index.ts

```diff
@@ -0,0 +1,7 @@
+import { expose, util } from '@chainlink/ea-bootstrap'
+import { makeExecute } from './adapter'
+import { makeConfig } from './config'
+
+const NAME = 'conflux'
+
+export = { NAME, makeExecute, makeConfig, ...expose(util.wrapExecute(makeExecute())) }
```

Comment:
We've made a change recently, `util.wrapExecute` won't be needed anymore. You can expose `makeExecute()` directly

Reply:
In my situation without `util.wrapExecute`, the `task run data` is empty, am i used a old version ea-bootstrap ?

Reply:
Update your branch with develop, and make sure you run `yarn setup`.
Please, restore your tests as well

Reply:
I have restore the tests, but it work on my local machine, failed on github actions, don't know why ?
```js
try {
      await execute(req.testData as AdapterRequest)
} catch (error) {
      const errorResp = Requester.errored(jobID, error)
      assertError({ expected: 400, actual: errorResp.statusCode }, errorResp, jobID)
}
```
The validate error code should be `400` right ?

Reply:
Another question is how to test successful calls ?
This need to send a real transaction on testnet, and how to get a valid request id ?